python/lsst/ts/MTMount/constants.py

```diff
@@ -18,6 +18,7 @@
 #
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <https://www.gnu.org/licenses/>.
+from lsst.ts.idl.enums.MTMount import AxisState, DriveState
 
 # Reply ports are one larger.
```

Comment:
As I commented in #36, I am weary of having these ports as constants. I understand that they will not change frequently, still, assuming they are "constants" seems like a bit of a stretch. I would rather have them as configurable parameters, but I understand if you want to leave them this way.